controllers/README.md

```diff
@@ -222,6 +222,27 @@ HTTP Response:
 }
 ```
 
+
+### Getting the responding to list of a userRespondingTo
+
+GET "/users/{id}/responding_to"
+
+HTTP Response:
+
+```
+{
+    "Responding_To": [
+        {
+            "username": string,
+            "id": string,
+            "naloxoneAvailability": boolean
```

Comment:
We don't need the availability field in the response

Reply:
@Asiapenguin Confirming we don't need the availability field in the response?

Reply:
We dont need it. The screens dont have availability status.

Reply:
Just an aside, thats not the right attribute for availability status anyway lol.

Reply:
got it lol